api/proto/node/node.proto

```diff
@@ -108,16 +108,19 @@ message Bundle {
 message BlobHeader {
 	// The KZG commitment to the polynomial representing the blob.
 	bytes commitment = 1;
```

Comment:
while we're making this breaking change, should we update the API so that it returns x and y coordinates of the commitment instead of the serialized bytes? 
same as `length_commitment` and `length_proof`

Reply:
Or JSON, it's much more widely accepted than gob

Reply:
yup either one 
@bxue-l2 if you want me to work on it let me know. I can work on a separate PR that this can rebase onto

Reply:
that will be great

Reply:
https://github.com/Layr-Labs/eigenda/pull/223